### PR TITLE
added newrelic integration

### PR DESCRIPTION
**Pull request by Dareen**

### CODE REVIEW

**Comment by Dareen** [comment]
@larikov @qasimkhan @dcrasto @iferminm @arahayrabedian can you guys please have a look

File: deploy/newrelic.ini [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+#
+
+[newrelic:development]
+monitor_mode = false
+
+[newrelic:test]
+monitor_mode = false
+
+[newrelic:staging]
+app_name = Flags (Staging)
```
Comment (larikov) [inline deploy/newrelic.ini]: Are you using different app names under the same newrelic account for production / staging envs or newrelic accounts are different and determined by newrelic license key?

**Comment by larikov** [comment]
looks good to me

File: deploy/newrelic.ini (hunk 1 above)
Comment (Dareen) [inline deploy/newrelic.ini]: different newrelic accounts for production/staging, the license key is an env var defined in EB based on the production/staging settings.
So, yes we will have different names for the app on production and on staging, and also in different newrelic accounts

**Pull request merged**